src/aragonjs-wrapper.js

```diff
@@ -438,18 +438,14 @@ const templateParamFilters = {
           `quorum (${minAcceptanceQuorum.toString()}) must be below 100%`
       )
     }
+    supportNeeded = supportNeeded.toString()
+    minAcceptanceQuorum = minAcceptanceQuorum.toString()
 
     const tokenBase = new BN(10).pow(new BN(18))
-    const holders = [{ address: account, balance: 1 }]
-
-    const [accounts, stakes] = holders.reduce(
-      ([accounts, stakes], holder) => [
-        [...accounts, holder.address],
-        [...stakes, tokenBase.muln(holder.balance)],
-      ],
-      [[], []]
-    )
+    const accounts = [account]
+    const stakes = accounts.map(() => tokenBase.toString())
 
+    // Note that we need all numerical arguments in string form for ABI encoding to work
```

Comment:
@bpierre This is mostly useful as a reference for the new onboarding; `web3@1` does not let you pass in BN.js numbers to contract invocations (or anything else requiring an ABI encode) anymore. We should always prefer serializing to a string before passing it off to an external API.